packages/backend/src/index.ts

```diff
@@ -1,25 +1,67 @@
+import * as dotenv from 'dotenv'
+
+import { resolve } from 'path'
+import { object, string } from 'zod'
+
+const envPath = resolve(__dirname, '../.env')
+
+const envVars = dotenv.config({ path: envPath })
+
+const envSchema = object({
+  PORT: string(),
+  DB_NAME: string(),
+  DB_USERNAME: string(),
+  DB_PASSWORD: string(),
+  JWT_ACCESS_TOKEN_SECRET: string(),
+  JWT_ACCESS_TOKEN_EXPIRATION_TIME: string(),
+  JWT_REFRESH_TOKEN_SECRET: string(),
+  JWT_REFRESH_TOKEN_EXPIRATION_TIME: string()
+})
```

Comment:
I'll probably move the environment variables to a separate/config file. I don't think we need `dotenv` if we are going to pass the environment variables through Docker.

Reply:
We can have a `config` folder where we would have `env.ts` and `logger.ts`.

Reply:
moved for now, but maybe we will have a talk about our environement variables.
Also, since knex.js is also a config, we might go back to also moving that into the new config folder, and referencing the file from package.json